lib/src/sql/parser.rs

```diff
@@ -52,41 +52,44 @@ fn parse_impl<O>(input: &str, parser: impl Fn(&str) -> IResult<&str, O>) -> Resu
 			// There was unparsed SQL remaining
 			Ok((_, _)) => Err(Error::QueryRemaining),
 			// There was an error when parsing the query
-			Err(Err::Error(e)) | Err(Err::Failure(e)) => Err(match e {
-				// There was a parsing error
-				Parser(e) => {
-					// Locate the parser position
-					let (s, l, c) = locate(input, e);
-					// Return the parser error
-					Error::InvalidQuery {
-						line: l,
-						char: c,
-						sql: s.to_string(),
+			Err(Err::Error(e)) | Err(Err::Failure(e)) => {
+				error!("parser error: {:?}", e);
```

Comment:
This line causes the entire block change - happy to revert, but I think it may be good to have it? We do intended to have improved parsing errors so it might not be necessary...